enhancements/console/internationalization.md

```diff
@@ -110,6 +110,15 @@ The desired fallback experience if not all pieces/parts of the UI are translated
 
 We will also modify our existing file conversion tools to handle translation files spread across multiple packages so we can easily export/import translation files.
 
+Translation of the login page will be handled by [oauth-server](https://github.com/openshift/oauth-server), which will check firstly the request URL for `lng` query. If the query is missing, `oauth-server` will check for request `Accept-Language` header. If both the query and header are missing in the request, `oauth-server` will differ to english language for localization.
```

Comment:
s/differ/default

Reply:
Why `lng` query **and** header? Why is the latter not enough?

Reply:
> Why `lng` query **and** header? Why is the latter not enough?

@sttts We need to be able to override the system setting since we have a user preference for selecting a language in console. If there is a standard query parameter, we can definitely use that instead.

I wouldn't drop the `Accept-Language` header because we want the default to be the system language when the user hasn't explicitly selected something different as a language preference in console.

Reply:
> I wouldn't drop the Accept-Language header because we want the default to be the system language when the user hasn't explicitly selected something different as a language preference in console.

That's fair. I would also expect the `ui_locales` to configure the session if set but I see no cookie handling here. Also, I am missing a description of the UI element that allows configuring this on the oauth-server side.

Reply:
Thats a good question. I think for now it has to be manual.

Reply:
I think console would store the last selected language in browser `localStorage` and then add `ui_locales` to the request when redirecting the user to login page. I haven't looked closely at the spec, though. Is this a query parameter?

Reply:
@spadgett thoughts?